docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3'
+services:
+  pep8speaks:
+    build: .
+    image: pep8speaks
```

Comment:
tag it to some version semantics eg. `pep8speaks:latest`

Reply:
This does not matter in local builds, planning to switch to GitHub packages all together soon, to avoid building images.